external-crates/move/crates/move-compiler/tests/move_check/parser/less_than_space.exp

```diff
@@ -2,8 +2,24 @@ error[E01002]: unexpected token
   ┌─ tests/move_check/parser/less_than_space.move:5:18
   │
 5 │         if (v< 10) return v;
-  │              - --^ Unexpected ')'
-  │              │ │  
-  │              │ Expected '::' after an address in a module access chain
-  │              Perhaps you need a blank space before this '<' operator?
+  │                --^ Unexpected ')'
+  │                │  
+  │                Expected '::' after an address in a module access chain
+
+error[E01002]: unexpected token
+  ┌─ tests/move_check/parser/less_than_space.move:5:28
+  │
+5 │         if (v< 10) return v;
+  │              -             ^ Expected '>'
+  │              │              
+  │              To match this '<'
+
+error[E01002]: unexpected token
+  ┌─ tests/move_check/parser/less_than_space.move:5:28
+  │
+5 │         if (v< 10) return v;
+  │                            ^
+  │                            │
+  │                            Unexpected ';'
+  │                            Expected ')'
```

Comment:
These errors suggest we need to do something smarter about parentheticals, but I appreciate that it is going to be difficult.